scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
+        fuel_dict['from_spent_mox'] = fuel_dict.pop('mox_mixer')
+        stacked_bar_chart(fuel_dict, timestep,
+                          'Years', 'Mass[MTHM]',
+                          'Total Fuel Mass vs Time',
+                          'where_fuel',
+                          init_year)
+
+        # get pu demand
+        demand = collections.OrderedDict()
+        demand['pu_from_legacy'] = [
+            i * .09 for i in fuel_dict['from_spent_uox']]
+        demand['pu_from_spent_mox'] = [
+            i * .09 for i in fuel_dict['from_spent_mox']]
+        total_mox = (
+            [x + y for x, y in zip(fuel_dict['from_spent_uox'], fuel_dict['from_spent_mox'])])
+        demand['pu_total'] = [i * .09 for i in total_mox]
+        demand['tailings'] = [i * .91 for i in total_mox]
+        multi_line_plot(demand, timestep,
                         'Years', 'Mass[MTHM]',
-                        'Total Waste Mass vs Time',
-                        'total_Waste',
+                        'Total Demand vs Time',
+                        'demand',
                         init_year)
 
-        fuel_dict = fuel_usage_timeseries(cur, ['uox', 'mox','fr_fuel'])
+        # get power / number timeseries
+        plot_power(cur)
 
+        # get reprocessing waste
+        reprocess_waste = commodity_in_out_facility(
+            cur, 'separations', ['mox_reprocess_waste', 'reprocess_waste'], True, False, False)
+        stacked_bar_chart(reprocess_waste, timestep, 'Years',
+                          'Mass [MTHM]', 'Reprocess Waste vs Time', 'reprocess_waste', init_year)
+
+"""
+        dictionary = collections.OrderedDict()
+        dictionary['natural_uranium'] = nat_u_timeseries(cur)
+        stacked_bar_chart(dictionary, timestep,
+                          'Years', 'U Utilization Factor',
+                          'U Utilization vs Time',
+                          'u_util', init_year)
+        combined = commodity_in_out_facility(cur, 'separations', ['mox_Pu'], True, False, True)
+        stacked_bar_chart(combined, timestep, 'Years', 'Mass[MTHM]', 'Total reprocessed Pu vs Time', 'combined', init_year)
+        mox_Pu = commodity_in_out_facility(cur, 'separations', ['mox_Pu'], True, False, True)
+        stacked_bar_chart(mox_Pu, timestep, 'Years', 'Mass[MTHM]', 'reprocessed Pu from spent MOX vs Time', 'reprocessed', init_year)
+        uox_Pu = commodity_in_out_facility(cur, 'separations', ['uox_Pu'], True, False, True)
+        stacked_bar_chart(uox_Pu, timestep, 'Years', 'Mass[MTHM]', 'reprocessed Pu from spent UOX vs Time', 'uox_reprocessed', init_year)
+        #stacked_bar_chart(get_trade_dict(cur, 'uox_reprocessing', 'mox_fuel_fab', True, True), timestep,
+        #                  'Years', 'Mass [MTHM]', 'reprocessed Pu outflux vs Time', 'ahhhhh', init_year)
+
+# Europe History Case Only
+        #tailings = commodity_in_out_facility(cur, 'uox_mixer', ['tailings'], True)
```

Comment:
This is bad practice. Make a separate function for this European History case if you must, but dont comment out large blocks of text.